pygrocy/grocy_api_client.py

```diff
@@ -40,6 +40,82 @@ def note(self) -> str:
     def amount(self) -> float:
         return self._amount
 
+class MealPlanResponse(object):
+    def __init__(self, parsed_json):
+        self._id = parse_int(parsed_json.get('id'))
+        self._day = parse_date(parsed_json.get('day'))
+        self._type = parsed_json.get('type')
+        self._recipe_id = parse_int(parsed_json.get('recipe_id'))
+        self._recipe_servings = parse_int(parsed_json.get('recipe_servings'))
+        self._note = parsed_json.get('note', None)
+        self._product_id = parsed_json.get('product_id')
+        self._product_amount = parse_float(parsed_json.get('product_amount'), 0)
+        self._product_qu_id = parsed_json.get('product_amount')
+        self._row_created_timestamp = parse_date(parsed_json.get('row_created_timestamp'))
+        self._userfields = parsed_json.get('userfields')
+
+    @property
+    def id(self) -> int:
+        return self._id
+
+    @property
+    def day(self) -> datetime:
+        return self._day
+
+    @property
+    def recipe_id(self) -> int:
+        return self._recipe_id
+
+    @property
+    def recipe_servings(self) -> int:
+        return self._recipe_servings
+
+    @property
+    def note(self) -> str:
+        return self._note
+
+
+class RecipeDetailsResponse(object):
+    def __init__(self, parsed_json):
+        self._id = parse_int(parsed_json.get('id'))
+        self._name = parsed_json.get('name')
+        self._description = parsed_json.get('description')
+        self._base_servings = parse_int(parsed_json.get('base_servings'))
+        self._desired_servings = parse_int(parsed_json.get('desired_servings'))
+        self._picture_file_name = parsed_json.get('picture_file_name')
+        self._row_created_timestamp = parse_date(parsed_json.get('row_created_timestamp'))
+        self._userfields = parsed_json.get('userfields')
+
+    @property
+    def id(self) -> int:
+        return self._id
+
+    def picture(self, width: int = 400):
```

Comment:
I'd like to have a more meaningful method name, maybe something like `get_full_picture_url`?